spec/reducers/config/networks/staticNetworks.spec.ts

```diff
@@ -0,0 +1,148 @@
+import { staticNetworks, makeExplorer } from 'reducers/config/networks/staticNetworks';
+import { ethPlorer, ETHTokenExplorer, SecureWalletName, InsecureWalletName } from 'config/data';
+import { StaticNetworkConfig } from 'types/network';
+import {
+  ETH_DEFAULT,
+  ETH_TREZOR,
+  ETH_LEDGER,
+  ETC_LEDGER,
+  ETC_TREZOR,
+  ETH_TESTNET,
+  EXP_DEFAULT,
+  UBQ_DEFAULT
+} from 'config/dpaths';
+
+const expectedInitialState = {
+  ETH: {
+    name: 'ETH',
+    unit: 'ETH',
+    chainId: 1,
+    isCustom: false,
+    color: '#0e97c0',
+    blockExplorer: makeExplorer('https://etherscan.io'),
+    tokenExplorer: {
+      name: ethPlorer,
+      address: ETHTokenExplorer
+    },
+    tokens: require('config/tokens/eth.json'),
+    contracts: require('config/contracts/eth.json'),
+    dPathFormats: {
```

Comment:
Same here, I think this could be imported.

Reply:
Same as above ^